installer/scripts/postinstall.sh

```diff
@@ -13,6 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+# if there is no python2, switch on python3. Since we passed preinstall, we think that at least python3 is available.
+if ! /usr/bin/env python2 --version >/dev/null; then
+  echo "Python2 was not found. Use python3."
```

Comment:
Perhaps should say something like "python2 binary not would, will use python3 binary for running the agent".

Reply:
Also, do we have a test case for this scenario?

To make sure that the agent is indeed correctly started after we switch the binary in this post install script.

Reply:
Thanks for pointing on that, I think I haven't paid enough attention to messages.

Reply:
> Also, do we have a test case for this scenario?

I think the test case - `test_<DISTRONAME>_python3` is responsible for that. All those tests call this common function where we check the python version after installation on the machine without python2 
https://github.com/scalyr/scalyr-agent-2/blob/342cb34efd1d320e159b6cefffeb6216ce10abaa/tests/distribution/python_version_change_tests/common.py#L36-L57